src/components/MainPage/Search.js

```diff
@@ -0,0 +1,105 @@
+import styled from "styled-components";
+
+
+const Search = styled.div`
+  grid-area: search;
+
+display:flex;
+align-items:center;
+
+flex-direction:column;
+
+
+/* Base styling */
+
+
+hr {
+  width:100%;
+  margin-top:20px;
+}
+
+button {
+margin-top:10px;
+  margin-right:2px;
+}
+
+.button-secondary {
```

Comment:
it seems you use lot of className in your project, i think you can do more component

Reply:
in progress for the next sprint